frontend/src/components/Footer/footer.module.css

```diff
@@ -0,0 +1,10 @@
+.footer {
+    background-color: #007ac9;
+    color: hsla(0,0%,94%,.95);
```

Comment:
Please use one type of colors system

Reply:
@IntiveMichal I switched to rgba everywhere :).